### ISSUE TITLE
general calrissian is not showing up as a "lando" for wild card during cloud city sabbacc

### ISSUE THREAD

**Issue by ericilvec** [PlayersCommittee/gemp-swccg-public#700]
- [ ] Did you search through the list of **open issues** _first_ to ensure that there is not already an report for this issue?

## **To Reproduce**
3. 
4.
5. See error

**Comment by EchoBaseTrooper** [comment]
Do you have a replay link?

**Comment by BOTVHD** [comment]
No notable changes in file history to:
- CC Sabacc (dark or light)
- General Calrissian
- PlaySabaccEffect

I suspect Dark had Cloud City: Casino in play, with the following game text on light side: "Unless you have a gambler here, you may not use wild cards in Cloud City Sabacc."  Light probably didn't have a gambler there and when CC Sabacc was played, General Calrissian was (correctly) not treated as a wild card.  This would be very easy to overlook, during a game.

I think this issue can probably be closed unless someone can provide a replay?

**Comment by Gergall** [comment]
Agreed. This normally works so in absence of a replay, have to assume it was the Casino, not a bug.
